src/api.js

```diff
@@ -87,4 +87,49 @@ export const MOCK_EVENT = {
 	yes_rsvp_count: 23
 };
 
+export const MOCK_GROUP_COMMUNICATION = {
+	"id":1342,
+	"description":"Some Mock Conversation",
+	"hashtag":"#ideas",
+	"group":MOCK_GROUP,
+	"creator":MOCK_MEMBER,
+	"num_comments":63,
+	"deleted":false,
+	"visible":true,
+	"created":1477520127,
+	"utc_offset":-18000000,
+	"updated":1477520127,
+	"self":{
+		"followed":true,
+		"actions":[
+			"delete",
+			"comment",
+			"follow",
+			"flag_spam"
+		]
+	}
+};
+
 
+export const MOCK_GROUP_COMMENTS = {
```

Comment:
something's not right here - this is identical to `MOCK_GROUP_COMMUNICATION`.

If possible, could you add a link to API docs if there are any?